src/ui/public/vis/vis.js

```diff
@@ -88,7 +87,6 @@ export function VisProvider(Private, Promise, indexPatterns, getAppState) {
         indexPatterns: indexPatterns,
         timeFilter: timefilter,
         queryFilter: queryFilter,
-        queryManager: queryManagerFactory(getAppState),
```

Comment:
Whoops! `vis.API.queryManager` is a documented Vis API feature: https://www.elastic.co/guide/en/kibana/current/development-vis-object.html Removing this broke integration for plugin authors.

Should this be re-added?